packages/eslint-plugin-next-on-pages/tests/rules/no-unsupported-configs.test.ts

```diff
@@ -463,4 +463,56 @@ describe('no-unsupported-configs', () => {
 			],
 		});
 	});
+
+	test('should work with .mjs and .cjs config file extensions', () => {
+		const validCode = `
+			/** @type {import('next').NextConfig} */
+			const nextConfig = {
+			}
+
+			module.exports = nextConfig
+		`;
+		const invalidCode = `
+			/** @type {import('next').NextConfig} */
+			const nextConfig = {
+				compress: true,
+			}
+
+			module.exports = nextConfig
+		`;
+		const invalidMessage =
+			'The "compress" configuration is not supported by next-on-pages (and is unlikely to be supported in the future).';
+		tester.run('', rule, {
+			valid: [
+				{
+					filename: 'next.config.mjs',
+					code: validCode,
```

Comment:
here you are running tests for the `next.config.mjs` file which contains the standard cjs code, could you please change the code to use esm? :pray:

Reply:
Done!